script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -149,7 +149,7 @@ Just like Arcueid said, these eyes and this life can't be anything but some kind
 Because ever since that day, I've been shown the end of the world.
 I've been shown the possibility that everything could fall apart in a second.
 As long as I'm alive, I'm surrounded by it.
-Filled to the brim by these portents of death.
+Drowning in these portents of death.//to be rechecked
```

Comment:
As long as I'm alive, I'm surrounded by **them**. These portents of death haunt me.

Reply:
> あんなにも、死の予兆が充満していた。

`Inundated by these portents of death`? `Smothered in these portents of death`?

Reply:
Smothered sounds good to me